tensorflow/lite/delegates/nnapi/java/src/main/java/org/tensorflow/lite/nnapi/NnApiDelegate.java

```diff
@@ -15,11 +15,14 @@
 
 package org.tensorflow.lite.nnapi;
 
+import java.io.Closeable;
```

Comment:
Nit: Let's use Autocloseable for consistency with the Interpreter. The GpuDelegate should have done the same, I'm updating it now.

Reply:
sounds good